charts/fluentd/values.yaml

```diff
@@ -374,3 +374,18 @@ fileConfigs:
         password changeme
       </match>
     </label>
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: []
```

Comment:
ident here: 
```yaml
ingress:
  hosts:
  - host: []
```

Reply:
As i said, if user sets ingress.enable and we dont have hosts it will be blank. Maybe set something as default? As you wish